.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+repos:
+
+- repo: https://github.com/psf/black
+  rev: 23.1.0
```

Comment:
Similar comment about matching what's in our action. If we're pinning a version here we likely need to pin the same version there.

Reply:
ill verify this